src/pages/recipes/{ContentfulRecipe.title}.tsx

```diff
@@ -1,74 +1,105 @@
 import React from 'react';
 
-import { graphql, PageProps } from 'gatsby';
-import { Helmet } from 'react-helmet';
+import { ChevronLeftIcon } from '@chakra-ui/icons';
+import { Flex, Heading, IconButton, Text, Image, HStack, VStack, Box } from '@chakra-ui/react';
+import { graphql, Link, PageProps } from 'gatsby';
+import TitledList from 'src/components/TitledList';
 
 import Layout from '../../components/Layout';
+import { parseRecipe } from '../../utils/parse';
 
 interface Props extends PageProps {
   data: GatsbyTypes.RecipeByTitleQuery;
 }
 
 function RecipeTemplate(props: Props): JSX.Element {
-  const recipe = props.data.contentfulRecipe;
-  const siteTitle = props.data.site?.siteMetadata?.title;
+  const contentfulRecipe = props.data.contentfulRecipe;
+  const recipe = parseRecipe(contentfulRecipe as GatsbyTypes.ContentfulRecipe);
 
   return (
     <Layout location={props.location}>
-      <div style={{ background: '#fff' }}>
-        <Helmet title={`${recipe?.title ?? ''} | ${siteTitle ?? ''}`} />
-        <div className="wrapper">
-          <h1 className="section-headline">{recipe?.title}</h1>
-          <h2>
-            Total Time: {recipe?.totalTime} {recipe?.totalTime == 1 ? 'min' : 'mins'}
-          </h2>
-          <h2>
-            Preparation Time: {recipe?.prepTime} {recipe?.prepTime == 1 ? 'min' : 'mins'}
-          </h2>
-          <h2
-            style={{
-              fontWeight: 'bold',
-            }}
+      <Box margin={{ base: 5, md: 10 }}>
+        <HStack display={{ base: 'none', md: 'flex' }} align="center" spacing={2} marginBottom={10}>
```

Comment:
If you like, you can use shorthand:
```suggestion
        <HStack display={{ base: 'none', md: 'flex' }} align="center" spacing={2} mb={10}>
```

Reply:
I personally like the full versions because its more readable for me but I can change all of them if that is the convention we want to follow.